runtime/drive_handler_test.go

```diff
@@ -33,7 +33,7 @@ import (
 )
 
 func TestStubDriveHandler(t *testing.T) {
-	tempPath, err := ioutil.TempDir("./", "TestStubDriveHandler")
+	tempPath, err := ioutil.TempDir("", "TestStubDriveHandler")
```

Comment:
Would we want to keep this behavior if people run the tests manually?

Reply:
I was wondering why we are using "./". That would be the reason.

How about asking people to manually set TMPDIR? Seems Go honors that.

* https://golang.org/pkg/io/ioutil/#TempDir
* https://golang.org/pkg/os/#TempDir

Reply:
Good point. I think it should be fine then.